### DISCUSSION TITLE
interactive tooltip

### DISCUSSION THREAD

**Discussion by rgarcia** [saadeghi/daisyui#3273] [Ideas / feature requests]
I'd like to have a tooltip that (1) stays open when you move the mouse over the tooltip. Use case is to allow users to copy / use the text contained in the tooltip.

Prior art: https://mui.com/material-ui/react-tooltip/#interactive

**Comment by saadeghi** [comment]
Consider using dropdown for interactive content:  
https://daisyui.com/components/dropdown/#open-on-hover

Tooltips are not for interactive content. It's something you would use similar to the HTML `title` attribute. The text can not be interacted with and does not capture the focus, because if it does, it would be impossible to interact what's under it.

Let me know if you have a question.

**Comment by rgarcia** [comment]
Thanks very much! TIL dropdowns can open on hover
